--- a/arch/arm64/boot/dts/freescale/Makefile	2026-03-01 22:10:48.559126410 +0100
+++ b/arch/arm64/boot/dts/freescale/Makefile	2026-03-01 22:11:35.653023868 +0100
@@ -17,6 +17,8 @@
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1043a-qds.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1043a-rdb.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1043a-tqmls1043a-mbls10xxa.dtb
+dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1043a-watchguard-t20t40.dtb
+dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1043a-watchguard-t20t40-w.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1046a-frwy.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1046a-qds.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1046a-rdb.dtb
